grails-app/taglib/org/pih/warehouse/SelectTagLib.groovy

```diff
@@ -490,7 +490,7 @@ class SelectTagLib {
 
             // use sparingly - this is expensive since it requires multiple database queries
             if (activityCode) {
-                attrs.from = attrs.from.findAll { it.supports(activityCode) || (!it.supports(activityCode) && it.supports(ActivityCode.SUBMIT_REQUEST)) }
+                attrs.from = attrs.from.findAll { it.supports(activityCode) }
```

Comment:
@kchelstowski have you ensured that requestors still can log into non-inventory managed locations?

Reply:
Oh now I see that it was "leftover" which was not removed